force-app/main/default/classes/testOpenTasksTrigger.cls

```diff
@@ -1,38 +1,100 @@
+/**
+ * @description       :
+ * @author            : Ferdi AKIN
+ * @last modified on  : 02-02-2023
+ * @last modified by  : Ferdi AKIN
+ **/
 @isTest
 public with sharing class testOpenTasksTrigger {
-    @isTest
-    private static void testCountOpenTasks() {
-        
-        Account testAccount = new Account(Name='Test Account', Type='Prospect', RecordTypeId=[SELECT Id FROM RecordType WHERE Name = 'United States' LIMIT 1].Id);
-        insert testAccount;
+  @isTest
+  private static void testCountOpenTasks() {
+    Account testAccount = new Account(
+      Name = 'Test Account',
+      Type = 'Prospect',
+      RecordTypeId = [
+        SELECT Id
+        FROM RecordType
+        WHERE Name = 'United States'
```

Comment:
Don't use a query, use the appropriate apex method
`
Schema.SObjectType.Account.getRecordTypeInfosByDeveloperName()`

Reply:
Also, dont use recordtype.name as name is not a unique value.